wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Commands.java

```diff
@@ -115,6 +118,17 @@ public static Command waitSeconds(double seconds) {
     return new WaitCommand(seconds);
   }
 
+  /**
+   * Constructs a command that does nothing, finishing after a specified duration.
+   *
+   * @param time after how long the command finishes
+   * @return the command
+   * @see WaitCommand
+   */
+  public static Command wait(Measure<Time> time) {
+    return new WaitCommand(time.in(Second));
```

Comment:
```suggestion
    return waitSeconds(time.in(Second))
```

Reply:
I would personally pass the measure here, letting the constructor do the conversion.

```suggestion
    return new WaitCommand(time);
```